src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/SdkFileDiffExclusions.txt

```diff
@@ -1,95 +1,95 @@
 # This list is processed using FileSystemName.MatchesSimpleExpression
 #
 # Format
-# {msft|sb},<path> [# comment]
-# msft = Microsoft built SDK
-# sb   = source-built SDK
+#   Exclude the path entirely: 
+#     <path> [# comment]
+#   Exclude a path from a specific sdk:
+#     <path>|{msft|sb} [# comment]
+#   msft = Microsoft built SDK
+#   sb   = source-built SDK
 #
 # Examples
```

Comment:
Would it make sense to document the common supported syntax in a md file and link it from all of the exclusion files?

Reply:
I think that this is a great idea! I can also expand on the allowed exclusions in that document.

Reply:
I'm wondering if this new document should exist in the SB documentation? I feel like that might be a better place for it.

Reply:
Not long term, at least. Because of https://github.com/dotnet/source-build/issues/4253, we're going to want to factor out this exclusions logic, and any associated documentation, out of SB-specific stuff.

Reply:
Makes sense. Maybe for now then, I'll leave the comments in the exclusions files and create a SB doc in a different PR. Then that doc will have to be moved out of SB later.